content/docs/user-guide/assets/scene-settings/motions-tab.md

```diff
@@ -0,0 +1,110 @@
+
+## Coordinate system change
+
+The Coordinate system change modifier is used to reorient a 3D scene source to Open 3D Engine's (O3DE) coordinate system. Most digital content creation (DCC) applications use a right-handed coordinate system with **Y** as the up axis. O3DE's coordinate system is left-handed with **Z** as the up axis.
+
+The Coordinate system change modifier has the following two modes:
+
+### Basic coordinate system change
+
+![The Scene Settings Meshes tab Coordinate system change modifier basic settings.](/images/user-guide/assets/scene-settings/coordinate-system-change-modifier-1.png)
+
+| Property | Description |
+| - | - |
+| **Facing direction** | Apply a 180 degree rotation around the up axis when product assets are processed. This property can be used to reorient the product assets to O3DE's forward axis, when the forward axis of the source asset aims toward the viewer (which is the default case in most DCC applications). |
+| **Use Advanced Settings** | When active, this property exposes the **Advanced coordinate system change** properties. |
+
+### Advanced coordinate system change
+
+![The Scene Settings Meshes tab Coordinate system change modifier advanced settings.](/images/user-guide/assets/scene-settings/coordinate-system-change-modifier-2.png)
+
+Change the **Translation** (position), **Rotation** (orientation), and **Scale** of a mesh relative to how it was authored.
+
+| Property | Description |
+| - | - |
+| **Relative Origin Node** | Select the source asset node that serves as the origin for the motion. The motion is transformed relative to the selected node. The node does not need to be part of the animated skeleton. Generally, the `RootNode` of the source asset is selected here as it represents the scene origin of the source asset. There are cases, however, where it might be preferable to apply the transform relative to a bone or mesh in the source file that is not the `RootNode`. |
+| **Translation** | Set the position offset of the product asset. This property is most commonly used to center the product asset or align the product asset with the ground.  |
+| **Rotation** | Set the orientation offset of the product asset in degrees. This property is most commonly used to reorient a product asset to O3DE's coordinate system. |
+| **Scale** | Set the scale offset of the product asset. O3DE's base unit is one meter. This property is most commonly used to uniformly scale product assets from DCC applications that use centimeters or inches as base units. |
+
+## Motion range
+
+![The Scene Settings Motions tab Motion range modifier.](/images/user-guide/assets/scene-settings/motion-range-modifier.png)
+
+Set the range of the animation for the motion. This is used to extract motions from source assets where several animation segments are included within a single frame range, one after another.
```

Comment:
Suggest adding info here about the start and end frame indexing. Afaik the animations in DCC tools are 1-indexed, **Start Frame** is 0-indexed and **End Frame** is non-inclusive. So a motion range of 0-36 includes motion frames 0-35 in the .motion asset, this range would have been indexed as frames 1-36 in a DCC tool.

Reply:
I can add a note for users to about this, but won't go into details for any specific DCC. I think most animators I've worked with tend to use a 0 frame in their DCC and key it to the bind pose or a rest pose.